### ISSUE TITLE
Minset recieve data - mapping seems strange

### ISSUE THREAD

**Issue by abrunstrom** [ietf-tapswg/api-drafts#822]
> Receive data (with no message delimiting): "Received" Event (Section 7.3.2.1). See Section 7.1.2 for handling Message framing
in situations where the Protocol Stack only provides a byte-stream transport.

Why is the part about Message framing here where we talk about receiving a byte stream rather than in the Receive message mapping?

**Comment by mwelzl** [comment]
I agree. PR #824 removes this sentence. The text talking about Receive message mapping already mentions Message Framers and points to Section 7.1.2.
